roles/nat64_appliance/molecule/default/converge.yml

```diff
@@ -108,6 +129,27 @@
         name: br-mol
         uri: 'qemu:///system'
 
+    # TODO(hjensas): With the zone set in libvirt XML these bridges "should"
```

Comment:
don't we need to also get that task in the actual role tasks? If we can't trust libvirt to push the interfaces in the right zone for molecule, we might face the same from standard usage?

Reply:
hmm, yeah I think you are right. Looking at my lab it is not adding the br-64v6 interface to the zone.

```
libvirt (active)
  target: ACCEPT                    
  icmp-block-inversion: no                                          
  interfaces: br-64v4 ironicendpoint ocpbm ocppr osp_trunk
```

I'll push an update ...